src/Components/AddRoom/AddRoom.tsx

```diff
@@ -0,0 +1,231 @@
+
```

Comment:
This component is alread included in the addRoom branch, ,you can consider removing it from here to avoid conflicts, or when merging accept the code coming from the addRoom branch.